maquetado/scss/tech.scss

```
@@ -19,11 +19,19 @@ $tech-logo-size: 100px;
     height: 320px;
     position: absolute;
     right: 10px;
+    transition: all .5s ease;
```

It's usually better to set multiple property transitions like so: 

```suggestion
transition: bottom .5s ease, right .5s ease;
```